bukuserver/views.py

```diff
@@ -563,7 +565,9 @@ def update_model(self, form, model):
             original_name = model.name
             form.populate_obj(model)
             self._on_model_change(form, model, False)
-            self.bukudb.replace_tag(original_name, [model.name])
+            names = {s for s in re.split(r'\s*,\s*', model.name.lower().strip()) if s}
+            assert names, 'Tag name cannot be blank.'  # deleting a tag should be done via a Delete button
+            self.bukudb.replace_tag(original_name, names)
```

Comment:
The `required` attribute cannot detect `,` as empty input (or `,, ,,` for that matter).

And if the user wanted to delete a tag, he'd be using the corresponding button.